docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -65,6 +65,11 @@ The main features of the Media Controller API include:
 
   You can [set the media controller server features](#media-controller-server-features) using the server interfaces.
 
+- Sending custom events
```

Comment:
Global comment:
Kindly ensure that all the codes are indented by 4 spaces.

Reply:
Im sorry but source code fragments in the mediacontroller guide are indented by 3 spaces everywhere. Cant do that.